apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -26,6 +26,10 @@
     <requiredProperty key="archetypeVersion">
       <defaultValue>${archetypeVersion}</defaultValue>
     </requiredProperty>
+    <requiredProperty key="apexVersion">
```

Comment:
apexCoreVersion? Should we allow to specify Malhar version as a property, instead of hardcoding it?